packages/experiments/src/components/PersonaCoin/PersonaCoin.view.tsx

```diff
@@ -25,12 +15,25 @@ export const PersonaCoinView: IPersonaCoinComponent['view'] = props => {
     root: 'div',
     image: PersonaCoinImage,
     initials: PersonaCoinInitials,
-    presence: PersonaPresence
+    presence: PersonaPresence,
+    personaCoinSize10: PersonaCoinSize10
   });
 
+  if (coinSize === 10) {
+    if (props.presence) {
+      // TODO: why do we need to pass size twice?
+      return <Slots.presence coinSize={coinSize} size={coinSize} />;
```

Comment:
Did you figure out this?

Reply:
Both are used for different parts of the sizing, but i'll leave this for now.
https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/Persona/PersonaPresence/PersonaPresence.base.tsx#L31-L59